ydb/library/yql/core/yql_opt_utils.h

```diff
@@ -95,6 +95,9 @@ TExprNode::TPtr MakeBool(TPositionHandle position, bool value, TExprContext& ctx
 TExprNode::TPtr MakeOptionalBool(TPositionHandle position, bool value, TExprContext& ctx);
 template <bool Bool>
 TExprNode::TPtr MakeBool(TPositionHandle position, TExprContext& ctx);
+TExprNode::TPtr MakePgBool(TPositionHandle position, bool value, TExprContext& ctx);
+template <bool Bool>
```

Comment:
bool can be passed as normal argument, please don't introduce a version with template bool

Reply:
I used the same approach as in MakeBool. Why MakePgBool should be different?